IDEAS/Buildings/Examples/ZoneCO2.mo

```diff
@@ -50,6 +50,15 @@ protected
   BoundaryConditions.WeatherData.Bus weaDatBus1
                             "Weather data bus connectable to weaBus connector from Buildings Library"
     annotation (Placement(transformation(extent={{-80,80},{-60,100}})));
+  Modelica.Blocks.Sources.RealExpression CEnv[max(Medium.nC,1)](y=sim.CEnv.y*
+        s_co2)
+    annotation (Placement(transformation(extent={{-62,78},{-44,94}})));
```

Comment:
@Mathadon I mean here. So as far as I understand you are using an auxiliar variable that if the substance is CO2 you add the CO2 from the environment. Shouldn't we add an example with a 'test' substance?

Reply:
For me that's not really required since we already describe how to use 'CO2' and thus adding the 'test' substance is trivial?

Reply:
@Mathadon Apart from this I do not have further remarks nor spotted errors. Therefore the model is ready to merge :)